src/main/kotlin/calculator/Expression.kt

```diff
@@ -0,0 +1,63 @@
+package calculator
+
+private const val COMMA = ","
+private const val COLON = ":"
+private const val FORWARD_SLASH = "//"
+private const val NEW_LINE_CHARACTER = "\n"
+
+class Expression(
+    private val text: String
+) {
+
+    fun isNumber(): Boolean {
+        val symbols = listOf(COMMA, COLON, FORWARD_SLASH, NEW_LINE_CHARACTER)
+        return symbols.none { text.contains(it) }
+    }
+
+    fun toPositiveInt(): Int {
+        val number = text.toIntOrNull() ?: throw RuntimeException("숫자가 아닙니다.")
+        validateMinusNumber(number)
+        return number
+    }
+
+    private fun validateMinusNumber(number: Int) {
+        if (number < 0) {
+            throw RuntimeException("음수는 입력할 수 없습니다.")
+        }
+    }
+
+    fun split(): List<Int> {
+        if (hasSymbol() && !hasCustomSymbol()) {
```

Comment:
```suggestion
        if (hasSymbol() && hasCustomSymbol().not()) {
```
자바에서는 !를 사용하기도하지만 코틀린에서는 boolean에 not을 제공하고 있어 좀 더 가독성있게 읽히는 코드가 작성이 됩니다.